lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -705,7 +731,10 @@ def write_request_via_grpc(entries:, log_name:, resource:, labels:)
                  error: error.to_s
     end
 
-    def write_request_via_rest(entries:, log_name:, resource:, labels:)
+    def write_request_via_rest(entries:,
```

Comment:
[Optional] If you rename this to just `write_request_rest`, the header will fit on one line.

Reply:
Hmm... I found it easier to read with parameters on separate lines especially when some of them have default values.